<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMonthlyCardDialog</class>
 <widget class="QDialog" name="CMonthlyCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>月租卡</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>254</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>有效期开始时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dtStart"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>254</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lblTilt1">
      <property name="text">
       <string>有效期结束时间：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dtEnd"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtComment"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>退出</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>30</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>卡号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtNumber"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>卡片自编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtSelfNumber"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>61</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>卡类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCardType">
      <item>
       <property name="text">
        <string>EMID卡</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MIFARE卡</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>登记者：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtRecorder">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>卡片使用状态：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCardStatus">
      <item>
       <property name="text">
        <string>启用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>挂失</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblStart">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkEnter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>152</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>入场必须刷卡</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkLeave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>152</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出场必须刷卡</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
